docs/docs/components/components-multichannel.md

```diff
@@ -26,52 +39,28 @@ As an example, it can be used to adapt native representations of a Text with But
                       
 
 
-## Example
+### Example
 
-Add the following code in your `action` files, under the `render` section.
+As an output, you would get something like this: 
 
 ```javascript
 <Multichannel
-  firstIndex={'a'}
-  boldIndex={true}
-  carousel={{ indexMode: 'letter' }}
-  text={{ indexMode: 'letter' }}
-  indexSeparator={'.'}
-  messageSeparator={'\n\n'}
->
-```
-
-### Description
-
-- `firstIndex` = The index is the option title of a choice. Ex: a. b. or 1. 2. etc.
-
-- `boldIndex`: Applies a bold format to the index element. True or False.
-
-- `Carousel`: Adds a letter or number to the index for a carousel.
-
-- `Text` : Adds a letter or number to the index for a text.
-
-- `Indexseparator`: Adds a dot or dash after the letter or number.
-
-- `Message separator` : Adds a space between the various messages. `{'\n\n'}` adds a line break between the index message and the selection option.
```

Comment:
Why removing descriptions for attributes?

Reply:
@vanbasten17 They are already listed in the Properties table, just the very same above the example. It's exactly the same content and see it as redundant.